### PR TITLE
Adding Qwicklabs Readme file

### PR DESCRIPTION
**Pull request by homernajafi**
Adding README file for QwickLabs

### CODE REVIEW

**Comment by chrislovecnm** [comment]
test this please

**Comment by homernajafi** [comment]
test this please

**Comment by homernajafi** [comment]
test this please

File: Jenkinsfile [hunk 1]
```diff
@@ -33,8 +33,7 @@ metadata:
 spec:
   containers:
   - name: k8s-node
-    image: gcr.io/pso-helmsman-cicd/jenkins-k8s-node:1.1.0
-    imagePullPolicy: Always
```
Comment (chrislovecnm) [inline Jenkinsfile]: We need the always tag please

File: Jenkinsfile [hunk 2]
```diff
@@ -65,7 +64,7 @@ spec:
   }
 
   environment {
-    CASSANDRA_VERSION = "3.11.3"
+    CASSANDRA_VERSION = "3.11.4"
```
Comment (chrislovecnm) [inline Jenkinsfile]: This exists? Do we need to push the new container?

**Comment by homernajafi** [comment]
test this please

File: Jenkinsfile (hunk 1 above)
Comment (chrislovecnm) [reply Jenkinsfile]: Can we get the image pull policy back please and I will merge.

Nice job fixing it!!

**Pull request merged**